Update overview-hyperrpc.md to remove extra closing braket (#497)
enviodev/docs: docs/HyperSync/HyperRPC/overview-hyperrpc.md

> - `eth_getLogs`
> - `trace_block` (only on [select chains](./hyperrpc-supported-networks))

> These methods are already supported on a wide selection of chains (the most up-to-date list can be found [here](./hyperrpc-supported-networks))

Reviewer: https://github.com/enviodev/docs/pull/499 a suggestion from a user in this PR is not to use "here" links for accessibility and a few other ux reasons.

```suggestion
These methods are already supported on a wide selection of chains  (This is [the most up-to-date list](./hyperrpc-supported-networks))
```
